src/multinomials.jl

```diff
@@ -3,6 +3,35 @@
 
 export multiexponents
 
+struct MultiExponents{T}
+  c::Combinations{T}
```

Comment:
4 space indents

Reply:
I always forget to adjust indentation on my editor, sorry, fixing it.